account_invoice_rounding/models/account.py

```diff
@@ -66,7 +66,7 @@ def _swedish_round_globally(self, invoice, rounded_total, delta):
         prec = obj_precision.precision_get('Account')
 
         ajust_line = None
-        for tax_line in invoice.invoice_line_ids.invoice_line_tax_ids:
+        for tax_line in invoice.tax_line_ids:
```

Comment:
I agree we have an issue on this line

This restore https://github.com/OCA/account-invoicing/blob/8.0/account_invoice_rounding/account.py#L88

We don't need to find the tax in the invoice lines but really the tax line on the invoice to adapt it.